new-log-viewer/src/utils/config.ts

```diff
@@ -88,9 +83,6 @@ const setConfig = ({key, value}: ConfigUpdate): Nullable<string> => {
                 value.timestampKey
             );
             break;
-        case CONFIG_KEY.THEME:
```

Comment:
Let's add some comment to explain why we're not handling `CONFIG_KEY.THEME` here.